README.md

```diff
@@ -128,6 +128,20 @@ Transforming fields requires two steps:
    end
    ```
 
+3. Deserializing *incoming* body params requires you add the
+   `JSONAPI.Deserializer` Plug to your API's pipeline. Note that the deserializer
+   expects the same casing for your *outgoing* params as your *incoming* params.
```

Comment:
I find this description a tad confusing. The way I understand this plug it sort of "flattens" the parts of a resource object to make it more convenient to use, especially when creating or updating records. The description, as is, has some overlap with the language used by the `UnderscoreParmeters` section, and has overlap with the sample code used by the `UnderscoreParamters` pipeline.

How about this:

> `JSONAPI.Deserializer` is a plug designed to make a JSON:API resource object more convenient to work with when creating or updating resources. This plug works by taking the resource object format and flattening it into an easier to manipulate Map.
>
> Your pipeline in a Phoenix app might look something like this:
> ```elixir
> pipeline :api do
>   plug JSONAPI.EnsureSpec # not required, but recommended for Deserializer
>   plug JSONAPI.Deserializer
> end
> ```

Reply:
Yes reading again and you are right!  Great suggestion!